images-list

```diff
@@ -78,6 +78,7 @@ flannelcni/flannel-cni-plugin rancher/mirrored-flannelcni-flannel-cni-plugin v1.
 flannelcni/flannel-cni-plugin rancher/mirrored-flannelcni-flannel-cni-plugin v1.1.0
 flannelcni/flannel-cni-plugin rancher/mirrored-flannelcni-flannel-cni-plugin v1.1.2
 flannelcni/flannel-cni-plugin rancher/mirrored-flannelcni-flannel-cni-plugin v1.2.0
+flannel/flannel rancher/mirrored-flannelcni-flannel v0.21.2
```

Comment:
If the source changes our mirrored name should change too, IMO. This will probably require getting EIO to create a new repo.
```suggestion
flannel/flannel rancher/mirrored-flannel-flannel v0.21.2
```

Reply:
good point I will make a request to EIO

Reply:
done